src/action/index.js

```diff
@@ -73,3 +73,9 @@ observe(store, 'lndReady', () => {
   transaction.subscribeTransactions();
   transaction.subscribeInvoices();
 });
+
+observe(store, 'syncedToChain', () => {
+  if (store.route == 'LoaderSyncing') {
+    nav.goHome();
+  }
+});
```

Comment:
If we put the navigation logic from the react component to an info action (see my comment below), we can do the observe there like in the wallet action: https://github.com/lightninglabs/lightning-app/blob/5545b3ee241a5df04a1894d328bfc583565c5b93/src/action/wallet.js#L130-L141